.github/workflows/php.yml

```diff
@@ -0,0 +1,64 @@
+name: PHP Composer
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
I thought this would only trigger the workflow on master. 

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#example-including-branches-and-tags

Reply:
After 0c9e702, I think this ought to trigger when pushing to `master` or when pushing to any pull request.